templates/web/merton/waste/garden/renew_bin_desc.html

```diff
@@ -0,0 +1,11 @@
+<div class="waste-service-hint-banner">
+    <div class="waste-service-hint-banner__image">
+        <img src="/i/waste-containers/bin-brown.png" alt="">
+    </div>
+    <div class="waste-service-hint__content">
+        <p>
+            Our garden waste bins are brown. They have a 240L
```

Comment:
So this is one of the things you need to expand on (e.g. including something for 140L)?